certified-connectors/Knowledgeone RecFind 6/apiDefinition.swagger.json

```diff
@@ -0,0 +1,376 @@
+            "name": "UserName",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "User Name",
+            "description": "User making the request"
+          },
+          {
+            "name": "QueryName",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Query Name",
+            "x-ms-summary": "Name of the predefined query"
+          },
+          {
+            "name": "SearchText",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Search Text",
+            "x-ms-summary": "Text to filter the search results"
+          }
+        ]
+      }
+    },
+    "/QueryData": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Records": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "RowCount": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "RowCount"
+                      },
+                      "Row_id": {
+                        "type": "string",
+                        "description": "Row_id"
+                      },
+                      "UrlView": {
+                        "type": "string",
+                        "description": "UrlView"
+                      },
+                      "UrlModify": {
+                        "type": "string",
+                        "description": "UrlModify"
+                      },
+                      "Row_fields": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "Field_name": {
+                              "type": "string",
+                              "description": "Field_name"
+                            },
+                            "Field_value": {
+                              "type": "string",
+                              "description": "Field_value"
+                            }
+                          }
+                        },
+                        "description": "Row_fields"
+                      }
+                    }
+                  },
+                  "description": "Records"
+                },
+                "LastPage": {
+                  "type": "boolean",
+                  "description": "LastPage"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result"
+                }
+              }
+            }
+          }
+        },
+        "summary": "QueryData",
```

Comment:
@microsoft/powershell-supermodule Please go thru the instructions mentioned in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions). All the summary and description values must be free from grammatical errors.

Reply:
Applies the same comment to all the similar summary values.